packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts

```diff
@@ -825,4 +830,95 @@ describe('JsTaskRunner', () => {
 			});
 		});
 	});
+
+	describe('idle timeout', () => {
```

Comment:
All tests pass, but there is now an additional Jest log that I haven't managed to remove. Ideas? 

```
Test Suites: 1 passed, 1 total
Tests:       430 passed, 430 total
Snapshots:   0 total
Time:        2.104 s, estimated 3 s
Ran all test suites matching /js-task-runner.test.ts/i.
  ●  process.exit called with "0"

      123 |
      124 |             this.idleTimer = setTimeout(() => {
    > 125 |                     if (this.runningTasks.size === 0) process.exit(0);
          |                                                               ^
      126 |             }, this.idleTimeout * 1000);
      127 |     }
      128 |

      at Timeout._onTimeout (src/task-runner.ts:125:46)
```

Reply:
I guess one way to do it would be to mock `process.exit` https://tsmx.net/jest-process-exit/

Reply:
Thank you, I moved to `EventEmitter` so asserting on the `emit` call now